Loop Simple Calculator.py

```diff
@@ -8,37 +8,27 @@ def simpcalc() :
 	op = input("Enter an operator: ")
 	
 	num2 = float(input("Enter your second number: "))
-		
-#Defining operators
-	addition = 1
-	subtraction = 1
-	division = 1
-	multiplication = 1
-	
```

Comment:
Those assignments do (almost) nothing because those variables are assigned values in the if-else statement below anyway.